Dockerfile

```diff
@@ -1,30 +1,36 @@
 FROM python:3.5.4-alpine
-ENV PYTHONUNBUFFERED 1
-
-# set timezone
-RUN apk update && apk add --no-cache tzdata && \
-    cp /usr/share/zoneinfo/America/Sao_Paulo /etc/localtime && \
-    echo "America/Sao_Paulo" >  /etc/timezone
-
-# pyscopg2 dependencies
-RUN apk update && apk add --no-cache \
-    gcc \
-    musl-dev \
-    postgresql-dev \
-    python3-dev
-
-# install git so we can pip install from github repos
-RUN apk update && apk add --no-cache git
-
-WORKDIR /code
+ENV PYTHONUNBUFFERED=1 \
+    GOSS_VERSION=${GOSS_VERSION:-v0.3.5}
 
 COPY ./requirements.txt /code/requirements.txt
 COPY ./requirements-dev.txt /code/requirements-dev.txt
-RUN python -m pip install -U pip && \
-    python -m pip install -r requirements-dev.txt
-
 COPY manage.py /code/manage.py
 COPY jarbas /code/jarbas
-RUN python manage.py collectstatic --no-input
+COPY goss.yaml /goss/goss.yaml
+
+WORKDIR /code
+
+# set timezone
+RUN set -ex && \
+    apk add --no-cache --virtual=.goss-dependencies curl ca-certificates && \
+    curl -L https://github.com/aelsabbahy/goss/releases/download/"$GOSS_VERSION"/goss-linux-amd64 -o /usr/local/bin/goss && \
+    chmod +rx /usr/local/bin/goss && \
+    apk update && apk add --no-cache tzdata libpq && \
+    cp /usr/share/zoneinfo/America/Sao_Paulo /etc/localtime && \
+    echo "America/Sao_Paulo" >  /etc/timezone && \
+    apk update && apk add --no-cache \
+      --virtual=.build-dependencies \
+      gcc \
+      musl-dev \
+      postgresql-dev \
+      git \
+      python3-dev && \
+    python -m pip --no-cache install -U pip && \
+    python -m pip --no-cache install -r requirements-dev.txt && \
+    python manage.py collectstatic --no-input && \
+    apk del --purge .build-dependencies .goss-dependencies
 
-CMD ["python", "manage.py", "runserver", "0.0.0.0:8000"]
\ No newline at end of file
+HEALTHCHECK --interval=1s --timeout=2m CMD goss -g /goss/goss.yaml validate
```

Comment:
Sore about my noob question: What benefits do we have from running health-check every second? Can we trigged some action if it fails? How does it work?

Reply:
That's the interval that the CMD will be run. Just to know faster if the container became unhealthy. We could set it to a greater time.

Reply:
Yep… I mean… 1s seems too much (Jarbas is already struggling in a bottle-neck in terms of hardware). In addition: this is thrown in the container logs, right?

Reply:
Yeah. Changed to 1m